src/components/card/videocard.jsx

```diff
@@ -0,0 +1,21 @@
+import "./videoCard.css"
+import "../../style/layout.css"
+export const VideoCard =(props)=>{
+    const data = props.singleVideoCard;
```

Comment:
if you only have one prop then you can do this:
instead of (props) use ({singleVideoCard : data})
this will give you the same result and will look better

Reply:
_Thank You, will do that_